app/src/CreateExcelFile/ExcelUtils.js

```diff
@@ -0,0 +1,23 @@
+import * as XLSX from "xlsx";
+
+function toExcelArray(searchItems) {
+  const result = searchItems.map((searchItem) => [
+    searchItem.title,
+    searchItem.link,
+  ]);
+  result.unshift(["Title", "Links"]);
+  return result;
+}
+
+export function createExcel(searchItemsList, searchQuery) {
```

Comment:
1. in 1 place ur using searchItems and in the other ur using searchItemsList
2. you are making it double plural here, searchItemSList. You either make it searchItems OR searchItemList